src/components/EditProfile/EditPersonalInformationForm.js

```diff
@@ -0,0 +1,321 @@
+                                            {selected.map((value) => (
+                                                <Chip key={value} label={value} />
+                                            ))}
+                                        </Box>
+                                    )}>
+                                    {allLanguages.map((language) => (
+                                        <MenuItem key={language} value={language}>
+                                            {language}
+                                        </MenuItem>
+                                    ))}
+                                </Select>
+                            </FormControl>
+                        </Grid>
+                        <Grid item xs={12}>
+                            <TextField
+                                id="outlined-multiline-static"
+                                label="Bio"
+                                multiline
+                                fullWidth
+                                rows={4}
+                                value={bio}
+                                placeholder="Tell everyone a little about yourself"
+                                inputProps={{ maxLength: 150 }}
+                                disabled={!props.editing}
+                                onChange={(e) => setBio(e.target.value)}
+                            />
+                        </Grid>
+                    </Grid>
+                    <Grid container item xs={12} sm={6} rowSpacing={2} sx={{ px: 2, pb: 2, display: "flex", alignContent: "flex-start"}}>
+                        <Grid item xs={12} md={8.7}>
+                            <TextField
+                                required
+                                id="email"
+                                name="email"
+                                label="Email"
+                                fullWidth
+                                value={email}
+                                autoComplete="email"
+                                variant="standard"
+                                disabled={!props.editing}
+                                onChange={(e) => setEmail(e.target.value)}/>
+                        </Grid>                    
+                        <Grid item xs={12} md={3.3} sx={{display: "flex", alignItems: "center", justifyContent: "flex-end"}}>
+                            <FormGroup>
+                                <FormControlLabel 
+                                    control={<Switch />} 
+                                    label="Public"
+                                    labelPlacement="start"
+                                    checked={emailPublic}
+                                    disabled={!props.editing}
+                                    onChange={(e) => setEmailPublic(e.target.checked)}
+                                />
+                            </FormGroup>                        
+                        </Grid>
```

Comment:
since all of the contact info fields are very similar, it may be worth making a Component that takes in a name, value, auto-complete type, checkedStatus, and onChange function to generate all these lines. not necessary since i don't think we'll be changing this page all that much, but it would make editing this section or adding a new type of contact less errorprone